third_party/gpus/crosstool/CROSSTOOL.tpl

```diff
@@ -121,6 +121,9 @@ toolchain {
   # linker_flag: "-Wl,--warn-execstack"
   # linker_flag: "-Wl,--detect-odr-violations"
 
+  # Include directory for cuda headers.
+  cxx_builtin_include_directory: "/usr/local/cuda%{cuda_version}/include"
```

Comment:
@damienmg is this sufficient? If not, I will still need to figure out what is the `nvcc` equivalent of `cc -E -xc++ - -v` to add an equivalent of [`_get_cxx_inc_directories`](https://github.com/bazelbuild/bazel/blob/master/tools/cpp/cc_configure.bzl#L125).